### DISCUSSION TITLE
how to add new words ?

### DISCUSSION THREAD

**Discussion by marwan211** [ArminJo/Talkie#26] [Q&A] [answered]
can you make a video tutorial to make talkie library speak specified words ?

**Comment by HectorOrdonez** [comment]
@marwan211 Did you figure this out eventually?

**Comment by ArminJo** [comment]
Not yet

**Comment by ahmedebeed555** [comment, marked as answer]
Finally I could generate voices. Thanks all for your help.
[https://youtu.be/KQseCA0nftI](https://youtu.be/KQseCA0nftI
)

**Reply by chache17** [reply]
The youtube link is broken. Can it be re-uploaded, please?
Are there any alternative guides?

**Comment by christurnerchris** [comment]
Yes it worked windows  incredible 
Just need a Linux version

**Comment by ArminJo** [comment]
@ahmedebeed555 I had no time to test it by myself, but I now added  a link to your video in the readme, many thanks 🥇

**Comment by christurnerchris** [comment]
tried it in  windows 10 . 
downloaded some samples from net - converted to wav using goldwave, 16 bit mono signed and 8bit mono both produce output but noise is mainly gibberish.  single piano notees just make terrible sound
Recorded my own voice (computer microphone) and quality is good.  think its background noise confuses process
amazing that anything works really !

**Comment by ahmedebeed555** [comment]
> @ahmedebeed555 I had no time to test it by myself, but I now added a link to your video in the readme, many thanks 🥇

Thank you very much.

**Comment by christurnerchris** [comment]
Got python_wizard_gui.py  to run in pyroid3 on Android
I don't know how to put the -f arduino on the end but it has defaulted to Arduino anyway!

**Comment by ISHIMWE-Aime** [comment]
Good morning; 
I am working on Ubuntu-based Linux 22.xx.xx, even though I have installed Python Wizard and scipy, by using this command `python3 python_wizard_gui -f arduino a.wav` the python wizard opens with the following parameters: 
`outputFormat: python` instead of Arduino, 
`tableVariant: tms5100`; 
I tried to change outputFormat manually to `arduino`, but it is generating terrible sounds. For tableVAriant, I don't know the chip I can specify. I tried manually to leave the parameter empty, but in the terminal, I got `KeyError: ' '`, so no generation of sound codes.
Anyone with support?
